util/src/main/i18n/templates/map.properties

```diff
@@ -54,6 +54,29 @@ map.setNext.confirm = You may not set the next map when a restart is queued. Use
 # {1} = name of map that was unset
 map.setNext.revert = {0} has removed {1} as the next map
 
+# {0} = player name (who added the map)
+# {1} = map name 
+map.setVote.addMap = {0} added {1} to the custom vote
```

Comment:
Let's change these keys to `vote.custom.add` etc (make them generic away from maps)

Reply:
Very good idea, looks much better now 😄